### DISCUSSION TITLE
How to use Mica and Acrylic for a Win32 on Windows 11

### DISCUSSION THREAD

**Discussion by mveril** [microsoft/WindowsAppSDK#1174] [Q&A]
How to use [Mica ](https://docs.microsoft.com/windows/apps/design/style/mica)and [Acrylic ](https://docs.microsoft.com/windows/apps/design/style/acrylic) for a Win32 (WPF for example) app on Windows 11 ?
I know that MS Edge Dev did it (for the titlebar and Flyout menus).
Maybe the new [DWMWINDOWATTRIBUTE](https://docs.microsoft.com/windows/win32/api/dwmapi/ne-dwmapi-dwmwindowattribute) `DWMWA_USE_HOSTBACKDROPBRUSH` is the key.
So if true, WinUI 3 acrylic microsoft/microsoft-ui-xaml#2236 issue can be easily resolved for Windows 11 (but continues to be an issue for backward compatibility).

**Comment by btueffers** [comment]
@nsanghai would you be able to provide some insight into this question?
